db/migration/1701450183524-TagTopicSlug.ts

```diff
@@ -0,0 +1,396 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+import fs from "fs/promises"
+import { get, set } from "lodash"
+
+/**
+ * This file includes the code I used to generate the data we're inserting into the new column
+ */
+
+// SELECT name FROM tags WHERE isTopic = TRUE
```

Comment:
I don't think any of this code needs to be reviewed. It's just here in case we need it again for some reason.